pkg/client/s3client.go

```diff
@@ -50,7 +50,6 @@ func NewS3Client(c *Client, instanceName, region string) (s3client *S3Client, er
 	var instanceID string
 	svcs, err := c.ResourceClientV2.ListInstances(controllerv2.ServiceInstanceQuery{
 		Type: "service_instance",
-		Name: instanceName,
```

Comment:
@bkhadars I see again you have removed these lines which got added in the previous fix

Reply:
@mkumatag because of the pagination issue, the tools fails to get the cos instance name, hence we were passing the cos instance name from the  command line. But with these we don't need Name option.